FIlter models created for Component

azizbek-abdurahmonov opened the pull request:
ComponentFilterDataModel and ComponentFilterModel created, Component model updated

Habiba67y commented on src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Application/Components/Models/Filters/ComponentFilterModel.cs:
```diff
@@ -0,0 +1,22 @@
+    public IEnumerable<ComponentCategory> Categories { get; set; }
+    public IEnumerable<Action> ListingTypes { get; set; }
+    public IEnumerable<string> Manufacturers { get; set; }
+    public int MinYear { get; set; }
```
FilterModel larda property lar nullable bo'lmaydimi?

Commit:
FIlter models created for Component

ComponentFilterDataModel and ComponentFilterModel created, Component model updated

Commit:
Fixed ComponentFilterModel

Mirzohid-Mominov reviewed (approved):
finished this one
